### PR TITLE
1565 notes hc eval update case note

### PR DESCRIPTION
**Pull request by MeganGeissler**
Discussed with Casey and working on incorportating the needed functionality.

### CODE REVIEW

**Review by IlseFerris — CHANGES_REQUESTED** [review]
Revw month not being pulled into case notes. Looks like a variable mismatch.

File: notes/health-care-evaluation.vbs [hunk 1]
```diff
@@ -2818,6 +2824,9 @@ function write_person_details_in_NOTE()
 		If HEALTH_CARE_MEMBERS(show_hc_detail_const, the_memb) = True Then
 			Call write_variable_in_CASE_NOTE("MEMB " & HEALTH_CARE_MEMBERS(ref_numb_const, the_memb) & " - " & HEALTH_CARE_MEMBERS(full_name_const, the_memb) & " - Processing: " & HEALTH_CARE_MEMBERS(HC_eval_process_const, the_memb))
 			Call write_variable_in_CASE_NOTE("     Status: " & HEALTH_CARE_MEMBERS(hc_eval_status, the_memb))
+			If trim(HEALTH_CARE_MEMBERS(HC_recertification_month, the_memb)) <> "" and trim(HEALTH_CARE_MEMBERS(HC_recertification_year, the_memb)) <> "" Then 
+				Call write_variable_in_CASE_NOTE("     HC Recertification Date: " & HEALTH_CARE_MEMBERS(HC_recertification_month, the_memb) & "/" & HEALTH_CARE_MEMBERS(HC_recertification_year, the_memb))
```
Comment (C-Love) [inline notes/health-care-evaluation.vbs:2828]: I am struggling a little with some of the convention here. 'HC Recert Date' reads a little weird to me. 

I think we could say 'HC Recertification Month:' or we could put the month in the 'Processing' section.

This is the current update:

========================== PERSON DETAILS ==========================
MEMB 01 - MONKEY LUFFY - Processing: Recertification                
     Status: Incomplete - other                                     
     HC Recertification Date: 01/24                                 
     Notes: This is a test case and I just want to see              

I think we can change it to this:
========================== PERSON DETAILS ==========================
MEMB 01 - MONKEY LUFFY - Processing: **01/24** Recertification                
     Status: Incomplete - other                                     
     HC Recertification Date: 01/24   **- Then remove this**                              
     Notes: This is a test case and I just want to see              

There is nothing wrong with it currently, we just typically refer to this as the recert month or MM/YY Revw/Recert.

**Review by C-Love — CHANGES_REQUESTED** [review]
Functionality looks good, I am wondering about some phrasing to make it more consistent with current communication around processing months and reviews. @courtrightd or @IlseFerris  - any thoughts on this? Maybe I am going down the rabbit hole a little.

File: notes/health-care-evaluation.vbs (hunk 1 above)
Comment (MeganGeissler) [reply notes/health-care-evaluation.vbs]: Revised case note to align with the second option above.

**Review by C-Love — APPROVED** [review]
Looks great!

**Pull request merged**